### PR TITLE
Documentation update for Azure Key Vault Task

### PR DESCRIPTION
**Pull request by degant**
- General Formatting and Code formatting (wherever required)
- Updating Secrets to Secrets + Certificates since Azure KV now provides access policies for both separately

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
take or close

File: Tasks/AzureKeyVault/README.md [hunk 1]
```diff
@@ -33,34 +31,38 @@ This task supports fetching latest values of the secrets which are already added
...
 
-Ensure the Azure endpoint has at least Get and List permissions on the vault. You can set these permissions in the Azure portal:
-Open the Settings blade for the vault, choose Access policies, then Add new.In the Add access policy blade, choose Select principal and select the service principal for your client account.In the Add access policy blade, choose Secret permissions and ensure that Get and List are checked (ticked). Choose OK to save the changes.
+Ensure the Azure endpoint has at least Get and List permissions for Secrets and Certificate Management on the vault. You can set these permissions in the Azure portal: Open the Settings blade for the vault, choose Access policies, then Add new. In the Add access policy blade, choose Select principal and select the service principal for your client account. In the Add access policy blade, choose `Secret permissions` and `Certificate permissions` and ensure that Get and List are checked (ticked) for both. Choose OK to save the changes.
```
Comment (lovakumar) [inline Tasks/AzureKeyVault/README.md]: This task handles only Secrets construct of Azure Key Vault. Azure has introduced Certificates construct recently hence we should not add certificates yet.

Comment (lovakumar) [inline Tasks/AzureKeyVault/README.md]: Remove Certificate permissions

**Comment by bansalaseem** [comment]
@degant, @lovakumar Can one of you please make the change that lova has requested?

**Pull request merged**